docs/GettingStarted/01-Why-Fedimint.md

```diff
@@ -22,37 +22,37 @@ Fedimint is built on three guiding pillars.
 
 Ideally bitcoiners should run their own nodes and custody their own funds.
```

Comment:
Personal preference but I like to just say bitcoin instead of funds, it's more explicit